README.md

```diff
@@ -77,4 +77,4 @@ may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
 Unless required by applicable law or agreed to in writing, software distributed
```

Comment:
Hey, I have a question: .....?

---
**Sema Reaction:** :question: I have a question

Reply:
The software uses external input to construct a pathname that should be within a restricted directory, but it does not properly neutralize sequences such as .. that can resolve to a location that is outside of that directory.

This is why... blah blah..

---
**Sema Reaction:** :question: I have a question